docs/limitations.md

```diff
@@ -1,12 +1,14 @@
 # Limitations
 
 <!-- Created by VSCode Markdown All in One command: Create Table of Contents -->
-- [StorageClass Reclaim Policy](#storageclass-reclaim-policy)
-- [Pod without PVC](#pod-without-pvc)
-- [Capacity Aware Scheduling May Go Wrong](#capacity-aware-scheduling-may-go-wrong)
-- [Snapshots Can Be Created Only for Thin Volumes](#snapshots-can-be-created-only-for-thin-volumes)
-- [Snapshots Can Be Restored Only on the Same Node with the Source Volume](#snapshots-can-be-restored-only-on-the-same-node-with-the-source-volume)
-- [Use lvcreate-options at Your Own Risk](#use-lvcreate-options-at-your-own-risk)
+- [Limitations](#limitations)
```

Comment:
We may not need this line because this listing is already in the `# Limitations` section.

Reply:
These lines are generated from the `Markdown All in One` plugin. The behavior of the plugin may have changed. To avoid this change, I added a marker.